### DISCUSSION TITLE
Chloropeth map in R

### DISCUSSION THREAD

**Discussion by eeinnew** [teamalgoritma/community#88] [Q&A] [answered]
Hello, I want to plot chloropeth map, actually I have finished plotting with leaflet but kinda interested in Chloropeth.
I have searched on the internet but still quite confused about this code and the function. (ex = layer and fortify)
<img width="960" alt="image" src="https://user-images.githubusercontent.com/81936292/167320008-0e957762-d10f-4f5d-a781-cb7f4afb90eb.png">

Here is the link :
https://medium.com/@anjesh/step-by-step-choropleth-map-in-r-a-case-of-mapping-nepal-7f62a84078d9

**Comment by amaliasf** [comment]
Hello, here is the reference related to the question that might help: [Creating Choropleth with Mapshaper and R](https://algotech.netlify.app/blog/creating-choropleth-with-mapshaper-and-r/)

**Reply by eeinnew** [reply]
hello Bu, from the link, I see that the data is in .JSON while mydata is in CSV
![image](https://user-images.githubusercontent.com/81936292/167342520-d154135e-5383-4c76-aee9-505f83c20a13.png)
will the implementation be the same? cause I get error in what = "sp"
![image](https://user-images.githubusercontent.com/81936292/167342579-9aec2933-dc7e-4f8a-b225-3ec454d67926.png)

**Reply by eeinnew** [reply]
can I skip to the leaflet step here?
![image](https://user-images.githubusercontent.com/81936292/167343342-57a0ca25-130d-47b7-89e0-85f2155f7b6c.png)

**Comment by amaliasf** [comment, marked as answer]
Hallo Bu Wennie @eeinnew , it is supposed to be fine when using csv file, no need to fill `what="sp"`

The last step is actually to make the choropleth, but we will get the same result via leaflet. If you are more familiar with leaflets, using that method will be easier. Hope this will help :)
